src/components/content/order/migrate/MigrateService.tsx

```diff
@@ -3,150 +3,127 @@
-        void ServicesAvailableService.listAvailableServices(categoryName, '', serviceName, serviceVersion).then(
-            (rsp: UserAvailableServiceVo[]) => {
-                if (rsp.length > 0) {
-                    setUserAvailableServiceVoList(rsp);
-                } else {
-                    return;
-                }
-            }
-        );
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [currentSelectedService]);
+    });
 
-    const getSelectedParameters = (
-        selectedCsp: CloudServiceProvider.name,
-        selectedArea: string,
-        selectedRegion: string,
-        selectedFlavor: string
-    ) => {
-        setSelectCsp(selectedCsp);
-        setSelectArea(selectedArea);
-        setSelectRegion(selectedRegion);
-        setSelectFlavor(selectedFlavor);
+    const prev = () => {
+        setCurrentMigrationStep(MigrationSteps.ImportServiceData);
+        getCurrentMigrationStep(MigrationSteps.ImportServiceData);
     };
 
-    const next = () => {
-        TipClear();
-        setIsTipShowStatus(false);
-        setCurrentMigrationStepStatus('process');
-        if (
-            currentMigrationStep + migrationStepsInterval ===
-            MigrationSteps.DeployServiceOnTheNewDestination.valueOf()
-        ) {
-            setIsNextDisabled(true);
-        }
-        setCurrentMigrationStep(currentMigrationStep + migrationStepsInterval);
+    const TipClear = () => {
+        setTip(undefined);
     };
 
-    const prev = () => {
-        TipClear();
-        setIsTipShowStatus(false);
-        setCurrentMigrationStepStatus('process');
-        if (
-            currentMigrationStep - migrationStepsInterval === MigrationSteps.SelectADestination.valueOf() ||
-            currentMigrationStep - migrationStepsInterval === MigrationSteps.ExportServiceData.valueOf()
-        ) {
-            setIsNextDisabled(false);
-        }
-        setCurrentMigrationStep(currentMigrationStep - migrationStepsInterval);
+    const migrate = () => {
+        setIsMigrating(true);
+        setIsPreviousDisabled(true);
+        startMigrating();
     };
 
-    const deploy = () => {
-        TipClear();
-        setIsTipShowStatus(false);
-        setCurrentMigrationStepStatus('process');
-        if (formRef.current) {
-            formRef.current.submit();
+    useEffect(() => {
+        if (currentMigrationStepStatus === undefined) {
+            return;
         }
-    };
-
-    function TipClear() {
-        setTip(undefined);
-    }
+        getCurrentMigrationStepStatus(currentMigrationStepStatus);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [currentMigrationStepStatus]);
 
-    const handleFinish = (values: Record<string, never>) => {
+    const startMigrating = () => {
+        if (currentSelectedService === undefined) {
+            return;
+        }
+        setCurrentMigrationStepStatus('process');
+        setIsDestroying(true);
```

Comment:
if i understood it right, 'isDestroying' status can be set true only after 'isDeployed' is true.  In all other cases, this value must be set to false.